cell_service! macro incompatible with zero-copy dispatch &Frame signature

## Issue

The `rapace_cell::cell_service!` macro still uses `&[u8]` for the dispatch signature, but after the zero-copy dispatch changes (commit c497030), the `ServiceDispatch` trait now expects `&Frame`.

## Reproduction

```rust
// In a cell implementation using the macro:
rapace_cell::cell_service!(
    MyServiceServer<MyServiceImpl>,
    MyServiceImpl
);
```

## Error

```
error[E0053]: method `dispatch` has an incompatible type for trait
  --> cells/cell-code-execution/src/main.rs:10:1
   |
10 | rapace_cell::cell_service!(CodeExecutorServer<CodeExecutorImpl>, CodeExecutorImpl);
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `Frame`, found `[u8]`
   |
   = note: expected signature `fn(&CellService, _, &Frame) -> Pin<_>`
              found signature `fn(&CellService, _, &[u8]) -> Pin<_>`
```

## Root Cause

The macro at `crates/rapace-cell/src/lib.rs:1066` needs to be updated from:

```rust
fn dispatch(
    &self,
    method_id: u32,
    payload: &[u8],  // ← Should be &Frame
) -> std::pin::Pin<...>
```

To match the `ServiceDispatch` trait which now expects `&Frame`.

## Context

This broke after updating rapace from a6a1e43 to c497030, which introduced the zero-copy dispatch optimization.

--- a/crates/rapace-cell/src/lib.rs
+++ b/crates/rapace-cell/src/lib.rs
@@ -937,6 +937,9 @@ macro_rules! run_cell_with_session {
 ///
 /// This is convenient when a proc-macro generates `FooServer<T>` where `FooServer::new(T)`
 /// constructs the server and `FooServer::dispatch(method_id, bytes)` routes calls.
+///
+/// The macro implements `ServiceDispatch` by extracting the payload bytes from the `Frame`
+/// and forwarding them to the server's dispatch method.
 #[macro_export]
 macro_rules! cell_service {
     ($server_type:ty, $impl_type:ty) => {
@@ -946,7 +949,7 @@ macro_rules! cell_service {
             fn dispatch(
                 &self,
                 method_id: u32,
-                payload: &[u8],
+                frame: &$crate::Frame,
             ) -> std::pin::Pin<
                 Box<
                     dyn std::future::Future<
@@ -956,7 +959,7 @@ macro_rules! cell_service {
                 >,
             > {
                 let server = self.0.clone();
-                let bytes = payload.to_vec();
+                let bytes = frame.payload_bytes().to_vec();
                 Box::pin(async move { server.dispatch(method_id, &bytes).await })
             }
         }

